test/e2e/api_transactions_create_test.go

```diff
@@ -303,6 +304,52 @@ var _ = Context("Ledger transactions create API tests", func() {
 						})
 					})
 				})
+				Context("using the `runtime` option to chose numscript rewrite", func() {
```

Comment:
This test does not prove the interpreter is called.
Maybe you could use a script with a feature only available in the interpreter?